content/css/concepts/display/display.md

```diff
@@ -13,7 +13,16 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Specifies the display behavior of an elements rendering box.
+# CSS Display Property
```

Comment:
```suggestion
```
We do not use h1 headers in Docs, nor do we have a title header. The Title field in the metadata takes care of that.